core/src/scss/elements/table/_table.scss

```diff
@@ -14,37 +14,41 @@
 // Style guide: Elements.Table
 //
 table {
-  @include margin(2em 0);
+  @include margin(2em 0 null 0);
```

Comment:
Should we remove margins outside of the element itself so we're consistent with all other Decanter components?

Reply:
I think consistency is a good principle to follow.